core/src/main/java/com/exadel/aem/toolkit/core/injectors/InjectorUtils.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exadel.aem.toolkit.core.injectors;
+
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ValueMap;
+
+/**
+ * Common methods for the injectors.
+ */
+public class InjectorUtils {
+
+    private InjectorUtils() {
+    }
+
+    /**
+     * Retrieves the {@code SlingHttpServletRequest}.
+     * @param adaptable the object which Sling tries to adapt from.
```

Comment:
(cosmetic) could you please format your javadocs the same way they are formatted in the rest of the project? 
the first letter of the param\return value description should be capitalized

Reply:
Yes, I changed it.